<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>623</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>781</width>
         <height>569</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>777</width>
         <height>569</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>799</width>
         <height>569</height>
        </size>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>801</width>
          <height>571</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="patients">
         <attribute name="title">
          <string>Patienten</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>9</y>
            <width>751</width>
            <height>151</height>
           </rect>
          </property>
          <property name="title">
           <string>Patienten Daten</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="2">
              <widget class="QComboBox" name="gender_dropdown">
               <item>
                <property name="text">
                 <string>male</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>female</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>divers</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="patient_id_field">
               <property name="maximumSize">
                <size>
                 <width>185</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="maxLength">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDateEdit" name="birthyear_calendar">
               <property name="dateTime">
                <datetime>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                 <year>2000</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
               <property name="displayFormat">
                <string>yyyy</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="ethnicity_dropdown">
               <item>
                <property name="text">
                 <string>American Indian or Alaska Native</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Asian</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Black or African American</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Native Hawaiian or Other Pacific Islander</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>White</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Other Race</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Enthnizität</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Jahr der ersten Symptome</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Geschlecht</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Jahr der Erstdiagnose</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Geburtsjahr</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Patienten-Id</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QDateEdit" name="firstdiagnosis_calendar">
               <property name="displayFormat">
                <string>yyyy</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDateEdit" name="firstsymptoms_calendar">
               <property name="displayFormat">
                <string>yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="4">
              <widget class="QPushButton" name="patient_select_button">
               <property name="text">
                <string>Auswählen</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="patient_delete_button">
               <property name="text">
                <string>Löschen</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="patient_search_button">
               <property name="text">
                <string>Suchen</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="patient_add_button">
               <property name="text">
                <string>Hinzufügen</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="patient_show_all_button">
               <property name="text">
                <string>Alle Anzeigen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>751</width>
            <height>371</height>
           </rect>
          </property>
          <property name="title">
           <string>Patienten Liste</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="previous_therapies">
         <attribute name="title">
          <string>Vortherapien</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="visits">
         <attribute name="title">
          <string>Visits</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
